billing/forms.py

```diff
@@ -26,17 +29,20 @@ class Meta:
             "business_address_line_4",
             "business_postcode",
             "business_tax_id",
+            "business_purchase_order",
         ]
         labels = {
             "account_name": "Account name",
             "business_name": "Business name",
             "business_tax_id": "Business VAT number",
+            "business_purchase_order": "Purchase Order",
         }
         help_texts = {
             "account_name": "A memorable name to identify this billing account. You will use this name whenever you make a booking to select which billing account it will be charged to.",
             "business_name": "The legal name of your business. This will be shown on your VAT invoices.",
             "business_address_line_1": "The address and post code provided here will appear on your VAT invoices.",
             "business_tax_id": "If you would like your VAT number shown on your invoices, please enter it here in the format GB123456789",
+            "business_purchase_order": "If you have a purchase order it will be displayed on your invoices, you may change this at any time.",
```

Comment:
```suggestion
            "business_purchase_order": "If you enter a purchase order number here it will be displayed on your invoices. You can change this at any time.",
```